docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/chip/chip.twig

```diff
@@ -5,94 +5,38 @@
-{% set size = size in ['extra-large', 'large', 'regular', 'small', 'extra-small'] ? size : 'regular' %}
-{% set icon_placement = icon_placement in ['before', 'after'] ? icon_placement : 'after' %}
-{% set icon_size = icon_size in ['regular', 'small', 'extra-small'] ? icon_size : 'regular' %}
 {% set theme_class = 'civictheme-theme-%s'|format(theme|default('light')) %}
-{% set type_class = 'civictheme-chip--chip' %}
+{% set kind = kind in ['default', 'input'] ? kind : 'default' %}
 {% set kind_class = kind ? 'civictheme-chip--%s'|format(kind) : '' %}
-{% set size_class = 'civictheme-chip--%s'|format(size) %}
-{% set selected_class = is_selected ? 'selected' : '' %}
-{% set dismiss_class =  is_dismissable ? 'civictheme-chip--dismiss' : '' %}
-{% set modifier_class = '%s %s %s %s %s %s %s'|format(theme_class, type_class, kind_class, size_class, selected_class, dismiss_class, modifier_class|default('')) %}
-
-{% set button_icon %}
-  {% if icon %}
-    <span class="civictheme-chip__icon">
-		  {% include '@atoms/icon/icon.twig' with {
-        symbol: icon,
-        size: icon_size,
-      } only %}
-		</span>
-  {% endif %}
-{% endset %}
-
-{% set button_text %}
-  {% if icon %}
-    {% if icon_placement == 'before' %}
-      {{ button_icon }}{{ is_raw_text ? text|raw : text }}
-    {% elseif icon_placement == 'after' %}
-      {{ is_raw_text ? text|raw : text }}{{ button_icon }}
-    {% endif %}
-  {% else %}
-    {{ is_raw_text ? text|raw : text }}
-  {% endif %}
-{% endset %}
+{% set multiple_class = is_multiple ? 'civictheme-chip--multiple' : '' %}
+{% set dismiss_class =  is_dismissible ? 'civictheme-chip--dismiss' : '' %}
+{% set modifier_class = '%s %s %s %s %s'|format(theme_class, kind_class, multiple_class, dismiss_class, modifier_class|default('')) %}
 
-{% set button_text %}
-  {% if is_external %}
-    {{ button_text }}<span class="civictheme-chip__external">
-    {% include '@atoms/icon/icon.twig' with {
-      symbol: 'upper-right-arrow',
-      size: icon_size,
-    } only %}
-    </span>
+{% set chip_content %}
+  {% if is_dismissible or kind == 'input' %}
+    {{ content }}<span class="civictheme-chip__dismiss" data-chip-dismiss>X</span>
   {% else %}
-    {{ button_text }}
+    {{ content }}
   {% endif %}
 {% endset %}
 
-{% set button_text %}
-  {% if is_dismissible %}
-    {{ button_text }}<span class="civictheme-chip__dismiss" data-button-dismiss>
-    {% include '@atoms/icon/icon.twig' with {
-      symbol: 'userinterface_cancel',
-      size: icon_size,
-    } only %}
-    </span>
+{% if content|trim is not empty %}
+  {% if kind == 'input' and is_multiple %}
+    <label class="civictheme-chip {{ modifier_class }}" data-component-name="chip"><input type="checkbox" value="{{ is_dismissible ?? 1 }}" {% if name is not empty %}name="{{ name }}"{% endif %} {% if attributes %}{{ attributes|raw }}{% endif %}>{{ chip_content|trim }}</label>
```

Comment:
please remove setting of the value for elements. the logic should be implemented outside of the component.

the ACs are stating the value of the `value` attribute after the click. But we should not be setting this value manually here.

Reply:
Removed